test/deploy_newnode.sh

```diff
@@ -115,7 +115,7 @@ mkdir -p $log_folder
 LOG_FILE=$log_folder/r.log
 
 HMY_OPT=
-HMY_OPT2=
+HMY_OPT2="-bootnodes /ip4/127.0.0.1/tcp/19876/p2p/QmWGSpah5QiBB1aKHKoJpM4VqJxcefc8kVxEbzXD59M8kP"
```

Comment:
how do you generate this?

Reply:
this shouldn't be hardcoded.  Please use $BN_MA.

Reply:
This is copied from the deploy.sh beacon chain output. I added a comment for it. And deleted my copy.